React/RequestParser.php

```diff
@@ -2,14 +2,13 @@
 
 namespace PHPPM\React;
 
-use React\Http\MultipartParser;
 use React\Http\Request;
 
-class RequestParser extends \React\Http\RequestParser
+class RequestParser extends \React\Http\RequestHeaderParser
 {
     public function parseHeaders($data)
     {
-        $request = parent::parseHeaders($data);
+        $request = parent::parseRequest($data);
```

Comment:
I guess we need to rename then the actual method as well, as `parent::` makes no sense otherwise.